### DISCUSSION TITLE
Adding Address for Meeting Location Issue

### DISCUSSION THREAD

**Discussion by centralofficeofwc** [code4recovery/12-step-meeting-list#486] [General]
Unable to Add Correct Address to Meeting Location for Frisco, CO AA Meeting

1. Go to https://aa-westerncolorado.com/meetings/?tsml-day=any&tsml-region=frisco
2. See error - City should be Frisco not Breckenridge

Correct Address should be 1002 N. 7th St. Frisco, CO 80443 not Breckenridge

Screenshots
https://user-images.githubusercontent.com/88400963/128070418-dd632be7-cff0-4684-938d-e64200d64b95.png
https://user-images.githubusercontent.com/88400963/128070536-69792bf7-b2f4-462c-8e00-42e009e1ac73.png

Desktop 
 - OS: Microsoft Windows 10
 - Browser Edge
 - Latest Version

**Comment by joshreisner** [comment]
we use google to do geocoding which is why you're seeing that city name. is the pin in the correct place?


![Screen Shot 2021-08-03 at 12 02 04 PM](https://user-images.githubusercontent.com/1551689/128071472-390cf59b-7aa0-475a-ad36-18304c9f25e3.png)

**Comment by joshreisner** [comment]
i saw your attached image - it seems the google 'places' api uses Frisco for the church, but the maps api uses Breckenridge for the address. hmm

**Comment by joshreisner** [comment]
We do have a way to override Google's answer for an address. If you have access to your WordPress theme you could add:

```
if (function_exists('tsml_custom_addresses')) {
    tsml_custom_addresses(array(
        '1002 N 7th Ave, Breckenridge, CO 80424, USA' => array(
            'formatted_address' => '1002 N 7th Ave, Frisco, CO 80424, USA',
            'city' => 'Frisco',
            'latitude' => 39.5666747,
            'longitude' =>  -106.0912529,
            'approximate' => 'no',
        ),
    ));
}
```
